vespa/src/main/java/ai/marqo/search/HybridSearcher.java

```diff
@@ -995,18 +995,33 @@ HitGroup postProcessResults(
             logIfVerbose("Applying global score modifiers and reranking.", verbose);
             resultToRerank = applyGlobalScoreModifiers(resultToRerank, query, verbose);
         } else if (query.properties().getBoolean("marqo__recency_apply_in_global_ranking_phase")) {
-            // apply recency score
+            // Apply recency score without global score modifiers
+            double addToScoreWeight =
```

Comment:
I am confused here. Why don't you put `marqo__recency_add_to_score_weight` in query.properties()?